.promu.yml

```diff
@@ -0,0 +1,25 @@
+go: 1.5.4
+repository:
+    path: github.com/prometheus/statsd_exporter
+build:
+    ldflags: |
+        -X main.Version={{.Version}}
+
+tarball:
+    files:
+        - LICENSE
+        - NOTICE
+crossbuild:
+    platforms:
+        - linux/amd64
+        - linux/386
+        - darwin/amd64
+        - darwin/386
+        - windows/amd64
+        - windows/386
+        - netbsd/amd64
+        - netbsd/386
```

Comment:
Took that from node-exporter. I've tried and they all can be built. Do we want to support more/less platforms?